packages/edit-navigation/src/components/menu-editor/use-navigation-blocks.js

```diff
@@ -88,19 +88,39 @@ function menuItemToLinkBlock(
 	innerBlocks = [],
 	existingBlock = null
 ) {
-	const attributes = {
-		label: menuItem.title.rendered,
-		url: menuItem.url,
-	};
+	let linkBlock;
```

Comment:
❓ `linkBlock` confused me later in the code, since it's not actually a Link block if it's a custom block, so maybe a more neutral name could do better like `menuBlock` and perhaps even rename the function as well?